Calibration/LumiAlCaRecoProducers/python/ALCARECOAlCaPCCRandom_cff.py

```diff
@@ -10,12 +10,14 @@
 )
 
 from EventFilter.SiPixelRawToDigi.SiPixelRawToDigi_cfi import siPixelDigis
-siPixelDigisForLumiR = siPixelDigis.clone()
-siPixelDigisForLumiR.InputLabel = cms.InputTag("hltFEDSelectorLumiPixels")
+siPixelDigisForLumiR = siPixelDigis.cpu.clone(
+    InputLabel = "hltFEDSelectorLumiPixels"
+)
```

Comment:
These (in all three ALCARECO files) should go together with the change of `siPixelDigis` to `SwitchProducer`.

Reply:
Thanks, I missed the `.cpu.` part ...